packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2426,12 +2437,14 @@
   "Livechat_last_chatted_agent_routing_Description": "The Last-Chatted Agent setting allocates chats to the agent who previously interacted with the same visitor if the agent is available when the chat starts.",
   "Livechat_managers": "Omnichannel managers",
   "Livechat_Managers": "Managers",
+  "Livechat_manual_on_hold_timeout": "How long an agent has to wait before placing the chat on-hold (in seconds), provided the agent has sent the last message",
```

Comment:
I think the setting title should be shorter and straightforward and use the setting description to share more details on how it works.

Reply:
This setting is no longer required